test/controllers/diary_entries_controller_test.rb

```diff
@@ -85,6 +85,10 @@ def test_routes
       { :path => "/user/username/diary/1/hide", :method => :post },
       { :controller => "diary_entries", :action => "hide", :display_name => "username", :id => "1" }
     )
+    assert_routing(
+      { :path => "/user/username/diary/1/unhide", :method => :post },
+      { :controller => "diary_entries", :action => "unhide", :display_name => "username", :id => "1" }
+    )
```

Comment:
Shouldn't there be a routing test for `unhidecomment` as well?

Reply:
Good catch